e2e-polybft/helpers_test.go

```diff
@@ -26,3 +28,66 @@ func (s *e2eStateProvider) Call(contractAddr ethgo.Address, input []byte, opts *
 func (s *e2eStateProvider) Txn(ethgo.Address, ethgo.Key, []byte) (contract.Txn, error) {
 	return nil, errors.New("send txn is not supported")
 }
+
+type validatorInfo struct {
+	address    ethgo.Address
+	rewards    *big.Int
+	totalStake *big.Int
+}
+
+// getRootchainValidators queries rootchain validator set
+func getRootchainValidators(relayer txrelayer.TxRelayer, checkpointManagerAddr, sender ethgo.Address) ([]*validatorInfo, error) {
+	getValidatorCount := func() (uint64, error) {
```

Comment:
You don't need this local function since you are using it in one place only.

Reply:
Yes you are 100% right, leftover :slightly_smiling_face: Nice finding! :100: